packages/core/src/domain/configuration/configuration.ts

```diff
@@ -14,30 +14,29 @@ export const DefaultPrivacyLevel = {
 export type DefaultPrivacyLevel = typeof DefaultPrivacyLevel[keyof typeof DefaultPrivacyLevel]
 
 export interface InitConfiguration {
+  // global options
   clientToken: string
-  applicationId?: string | undefined
-  actionNameAttribute?: string | undefined
-  internalMonitoringApiKey?: string | undefined
-  allowedTracingOrigins?: Array<string | RegExp> | undefined
+  beforeSend?: BeforeSendCallback | undefined
```

Comment:
do we need to keep these common `beforeSend` and `BeforeSendCallback`?
cf #822

Reply:
Mmh I can move it. The downside is that we'll need to have the following code in both RUM and Logs configuration build:
```ts
    beforeSend:
      initConfiguration.beforeSend && catchUserErrors(initConfiguration.beforeSend, 'beforeSend threw an error:'),
```
but I guess that's fine

Reply:
indeed.
Alternate proposal: keep the code here but make more clear here that it is a more generic type with real implementation in logs / rum.
it should avoid the confusion of the related ticket.
wdyt?